### DISCUSSION TITLE
Prefix or postfix dollar sign ($) to access signal value like Svelte with stores

### DISCUSSION THREAD

**Discussion by ruggi99** [solidjs/solid#1479] [Ideas]
It is only a syntactic sugar but I think it will make cleaner than accessor functions, expecially when signals are inside other parenthesys.
Eventually an underscore could do the trick.
Personally I'd prefer a postfix $ to access signal value.

Before:

```javascript
function MyComponent() {
  const [signal, setSignal] = createSignal(0);

  return <div>{signal()}</div>
}
```
After
```javascript
function MyComponent() {
  const [signal, setSignal] = createSignal(0);

  return <div>{signal$}</div>
}
```

I think it should only transpile from `signal$` to`signal()`
Don't know if it is easy to implement in Typescript

**Comment by ryansolid** [comment]
Yeah TS wouldn't be your friend. You might be interested in this project with similar syntax built on Solid: https://github.com/orenelbaum/babel-plugin-reactivars-solid

Honestly I have no interest messing with this. The fact accessors are just functions is a superpower. If there was a way to get rid of the function calls that'd be something but replacing JavaScript for an arbitrary syntax is asking for downstream complexity with very little benefit.
